pebbles/drivers/provisioning/openshift_driver.py

```diff
@@ -600,6 +593,17 @@ def _do_deprovision(self, token, instance_id):
             else:
                 raise RuntimeError(res.reason)
 
+        # then set replicas to 0 and let pods die
+        params = dict(labelSelector='run=%s' % name)
+        rc_list = oc.search_by_label(
```

Comment:
Do not understand what happened here, what are the different params passed to get the list. Maybe a bit more explanation ?